sdk/ai/azure-ai-projects/samples/evaluations/sample_evaluations_schedules.py

```diff
@@ -1,81 +1,191 @@
-from azure.ai.projects import AIProjectClient
-
-from azure.identity import DefaultAzureCredential
-from azure.ai.projects.models import (
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+Installation Instructions
+
+pip install azure-identity azure-ai-projects azure-ai-ml
+
+# Optionally, you can `pip install azure-ai-evaluation` if you want a code-first experience to fetch evaluator id for built-in evaluators in code
+# refer https://learn.microsoft.com/en-us/azure/ai-studio/how-to/online-evaluation for more details
+"""
+import pprint
+from azure.ai.projects import AIProjectClient 
+from azure.identity import DefaultAzureCredential 
+from azure.ai.projects.models import ( 
     ApplicationInsightsConfiguration,
     EvaluatorConfiguration,
     EvaluationSchedule,
-    CronTrigger,
     RecurrenceTrigger,
-    Frequency,
-    RecurrenceSchedule,
+    Frequency
+)
+from azure.ai.evaluation import CoherenceEvaluator  
+
+# This sample includes the setup for an online evaluation schedule using the Azure AI Project SDK and Azure AI Evaluation SDK
+# The schedule is configured to run daily over the collected trace data while running two evaluators: CoherenceEvaluator and RelevanceEvaluator
+# This sample can be modified to fit your application's requirements
+
+# Name of your online evaluation schedule
+SAMPLE_NAME = "online_eval_name"
```

Comment:
Our samples read configuration from environment variables, so we can easily run samples without modifying source code. See other samples. Please consider doing the same here.

Reply:
@dargilco I agree that if a variable remains constant across runs of the sample, it’s better to make it an environment variable. However, in this sample, these variables (resource names) are intended to be changed frequently. Therefore, we have made them fixed variables in the file.